### DISCUSSION TITLE
SVG Display in Readme

### DISCUSSION THREAD

**Discussion by hugodemenez** [community/community#4103] [Mobile]
On the github iOS mobile app, I can't visualize any svg img present in the readme.md. 

Still not fixed on GitHub app version : 1.124.0

Are there options to activate ? 

Sincerely, 
Hugo

**Comment by fayasnoushad** [comment]


**Reply by hugodemenez** [reply]
The issue is on the mobile app. It doesn't display the svg file on mobile. Indeed, the svg is visible on computer or in the mobile webbrowser. Thank you for the response.

**Reply by bxbdbdbj** [reply]
> Search a svg picture in google and copy the html url of image and add your readme.md

**Reply by bxbdbdbj** [reply]
> The issue is on the mobile app. It doesn't display the svg file on mobile. Indeed, the svg is visible on computer or in the mobile webbrowser. Thank you for the response.

**Reply by bxbdbdbj** [reply]
> > The issue is on the mobile app. It doesn't display the svg file on mobile. Indeed, the svg is visible on computer or in the mobile webbrowser. Thank you for the response. 
> 
>

**Comment by aconsuegra** [comment]
Hi @hugodemenez. Can you tell me in which platform (Android/iOS) are you having issues to display SVGs?

**Reply by hugodemenez** [reply]
I am on IOS 14.6 on IPhone 11 Pro.

**Reply by hugodemenez** [reply]
GitHub app version :1.17.0

**Reply by aconsuegra** [reply]
Thanks @hugodemenez! Do you have a link to a repository where you are experiencing this issue with SVGs in readme files? I'm having troubles to reproduce the issue myself.

**Reply by erbanku** [reply]
@aconsuegra The issue still exists in iOS latest GitHub mobile app

**Reply by Vannsansan** [reply]
Hi

**Comment by fedorkanan** [comment]
hi dear Hugo Demenez
try to use css properties in the html tag that you use in your about page like : 
width
height 
align="cnter"
......                             
but 
when it happen in only one device try to clear cache
and reboot the app if it's not working then do this :
use see the mobile site in pc or on the mobile website see what you got

**Reply by hugodemenez** [reply]
Hi,
Thanks for your tips.

In fact, the issue is only visible on the iOS mobile app.

If I use the mobile browser the issue doesn’t remain.

De facto, all your tips are working 😉 but that’s not the point of the raised issue. 

Thanks anyway !
